ctci/chapter2/2.4.py

```diff
@@ -0,0 +1,76 @@
+#2.4
+#Partition: Write code to partition a linked list around a value x, 
+#such that all nodes less than x come before all nodes greater than 
+#or equal to x. lf x is contained within the list, the values of x only
+#need to be after the elements less than x (see below).The partition element 
+#x can appear anywhere in the "right partition"; it does not need to appear between the left and right partitions.
+
+#assuming no cycles
+
+from random import randrange
+
+class Node:
```

Comment:
For my leetcode stuff I have one class defining a Node + LinkedList + Tree and just import that across all my files, that possible in python or too much of a hassle?